### ISSUE TITLE
Trying to print a popup window with programmatically created content crashes WebView2

### ISSUE THREAD

**Issue by RichardSteele** [MicrosoftEdge/WebViewFeedback#134]
Executing following code causes WebView2 to crash, regardless of having a handler attached to the NewWindowRequested event:

```
var popup = window.open();
popup.document.write('Hello World');
popup.print();
```

**Comment by david-risney** [comment]
Great find, thanks! We'll take a look at it.

**Comment by champnic** [comment]
This has been fixed and should be available in Edge 83.0.464.0+. Thanks!

**Comment by RichardSteele** [comment]
It works, but only if you let WebView2 handle the popup. I have opened follow-up issue #293.
